src/quick-lane/views/QuickLaneDetail.tsx

```diff
@@ -205,11 +212,14 @@ const QuickLaneDetail: FunctionComponent<QuickLaneDetailProps> = ({
 				);
 			case 'ITEM':
 				return (
-					<ItemVideoDescription
-						itemMetaData={quickLane.content as Avo.Item.Item}
-						showDescription={contentLayout === AssignmentLayout.PlayerAndText}
-						verticalLayout={isMobileWidth()}
-					/>
+					<>
+						<ItemVideoDescription
+							itemMetaData={quickLane.content as Avo.Item.Item}
+							showDescription={contentLayout === AssignmentLayout.PlayerAndText}
+							verticalLayout={isMobileWidth()}
+							cuePointsLabel={{ start, end }}
+						/>
+					</>
```

Comment:
do we need the fragment wrapper <></> here?

Reply:
Nope, will remove